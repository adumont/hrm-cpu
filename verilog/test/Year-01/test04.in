08 65
24 e7
17 0a
35 ae
