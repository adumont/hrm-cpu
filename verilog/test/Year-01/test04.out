65
e7
0a
